pages/index.tsx

```diff
@@ -46,7 +50,60 @@ export default function Home() {
         height={85}
       />
       <InfoColumns />
-      <ComponentWrapper margins={"auto"} pad={"0 10rem 0 10rem"}>
+      <ComponentWrapper margins={"auto"} pad={"0rem 6rem 0rem 6rem"}>
```

Comment:
can this be it's own component so each page looks clean with minimal styling ie

<Wrapper>
    <Jumbotron />
    <InfoColumns />
    <Letter />
    <Footer />
</Wrapper>


#composition4dawin